sdk/dotnet/As3.cs

```diff
@@ -0,0 +1,114 @@
+// *** WARNING: this file was generated by the Pulumi Terraform Bridge (tfgen) Tool. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.F5BigIP
+{
+    /// <summary>
+    /// `f5bigip..As3` provides details about bigip as3 resource
```

Comment:
It looks like we have a small glitch in handling "Index" modules (`f5bigip..As3`)